about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <section class="header-section">
+        <div class="main-containers">
+            <nav class="inner-header">
+                <li class="inner-list"><img src="images/facebook.png" alt=""></li>
+                <li class="inner-list"><img src="images/twitter.png" alt=""></li>
+                <li class="inner-list"><a href="#">English</a></li>
+                <li class="inner-list"><a href="#">My page</a></li>
+                <li class="inner-list"><a href="#">Logout</a></li>
+            </nav>
```

Comment:
- [ ] Kindly follow the given design by removing the top dark navigation and the logo from your mobile view design.

Given design
![image](https://user-images.githubusercontent.com/81932478/196734327-c9806c8c-b0ad-4892-a461-d673bb7258c4.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/196734467-bdbb323e-0431-4cb9-8d2b-ccb39066b194.png)